<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>About me - Roland Faure</title>


    <!-- web-fonts -->
    <link href="https://fonts.googleapis.com/css?family=Hind:300,400,500,600,700" rel="stylesheet">

    <!-- font-awesome -->
    <link href="css/font-awesome.min.css" rel="stylesheet">

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- owl carousal -->
    <link href="css/owl.carousel.css" rel="stylesheet">

    <!-- Style CSS -->
    <link href="css/style.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<body>
<div id="main-wrapper">
<!-- Page Preloader -->
<div id="preloader">
    <div id="status">
        <div class="status-mes"></div>
    </div>
</div>

<header class="header">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="profile-img">
                    <img src="img/img-profile.png" class="img-responsive" alt=""/>
                </div>
                <!-- Profile Image -->
            </div>
            <div class="col-md-9">
                <div class="name-wrapper">
                    <h1 class="name">Roland Faure</h1>
                </div>
                <p>
                    I'm a PhD student in bioinformatics since October 2021 under the joint supervision of <a href="http://lavenier.net/homepage/"> Dominique Lavenier </a> in the <a href="https://team.inria.fr/genscale/"> Genscale team in Rennes, France </a> and <a href="https://ebe.ulb.ac.be/ebe/Flot.html"> Jean-François Flot </a> and the EBE team at the Université Libre de Bruxelles, Belgium. I like imagining and implementing algorithmic solutions on biological problems. My area of research is genome assembly, more specifically the problems of haplotyping and metagenome assembly. The main work of my Ph.D. is the development of the tool <i>HairSplitter</i>, which recovers in an assembly strains that have been missed by an assembler (generally because they were too similar to other strains).
                </p>


      
            </div>
        </div>
    </div>
</header>
<!-- .header-->

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Software, publications & talks</h2></div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Hairsplitter</h3>
                            <h4>Separating noisy long reads into an unknown number of haplotypes</h4>
                            
			    <p><i>Developped with the help of Dominique Lavenier and Jean-François Flot </i></p>

                            <p>
                            	The goal of Hairsplitter is to produce contiguous and uncollapsed (i.e. with all haplotypes still present) (meta-)genome assembly. It takes an assembly and sequencing reads as input, detects if contigs have been collapsed and output the corresponding uncollapsed assembly. Written in C++. <a href="https://github.com/RolandFaure/Hairsplitter" > Available on GitHub.</a> The most complete paper yet can be found on the <a href="https://jobim2023.sciencesconf.org/data/pages/proceedings.pdf">JOBIM 2023 proceedings</a>.<br>
                            	Presented as a poster at <a href="https://coursesandconferences.wellcomeconnectingscience.org/event/genome-informatics-20220921/"> Genome informatics 2022</a> and <a href=http://iscb.org/ismbeccb2023> ISMB/ECCB 2023 </a>. <a href=https://github.com/RolandFaure> See the poster</a>. Presented as a talk at <a href="https://seqbim.cnrs.fr/"> SeqBIM 2022</a>, <a href=https://jobim2023.sciencesconf.org> JOBIM 2023 </a> and <a href=http://iscb.org/ismbeccb2023> ISMB/ECCB 2023 </a>.
                            </p>

                        </div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Graphunzip</h3>
                            <h4>Untangling assembly graphs to finish assemblies</h4>
                            <p><i>Developped under the supervision of Nadège Guiglielmoni and Jean-François Flot </i></p>

                            <p>
                            	GraphUnzip untangles assembly graphs with the help of long reads and/or Hi-C. It is designed to distinguish the different haplotypes present. Written in Python. <a href="https://github.com/nadegeguiglielmoni/GraphUnzip" > Available on GitHub.</a><br>
                            	Presented the key idea for the first time at <a href="https://jobim2021.sciencesconf.org/"> JOBIM 2021 </a> <br>
                            	The long read algorithm was presented for the first time at <a href="https://seqbim.cnrs.fr/"> SeqBIM </a> 2021 thanks to a grant from the SFBI.
                            </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>QuickDeconvolution</h3>
                            <h4>Rapidly separating barcoded reads into groups</h4>
                            
			    <p><i>Developped under the supervision of Dominique Lavenier </i></p>

                            <p>
                            	QuickDeconvolution aims at answering the barcode deconvolution problem when dealing with barcoded reads. It strives at being scalable and parallelisable. Written in C++. <a href="https://github.com/RolandFaure/QuickDeconvolution" > Available on GitHub.</a><br>
                            	<a href="https://academic.oup.com/bioinformaticsadvances/advance-article-abstract/doi/10.1093/bioadv/vbac068/6717790"> Published </a> in <i>Bioinformatics advances</i> <br>
                            </p>

                        </div>
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Reviews</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>NAR Genomics and Bioinformatics</h3>

                            <p>
                            	Co-reviewed a paper with Karel Brinda, october 2022
                            </p>
                          
                        </div>
            </div>
            
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Students</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Tam Minh Khac Truong </h3>
				<h4>StrainMiner: Data mining and discrete optimizations for strains separation in metagenomes using long reads</h4>
				<p>Two masters internships of 4 months (2022) and 6 months (2023). Co-supervised with Rumen Andonov <br>
                            	Worked on a new approach to separate error-prone sequencing reads from several strains of the same species in metagenome. The new approach is based on integer linear programming and showed good performance in some cases. Hopefully published soon. </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Olivier De Thier</h3>
                            <h4>Improving ascidian de novo genome assemblies with long reads</h4>
                            <p>Master thesis of 6 months (2023). Co-supervised with Jean-François Flot and Stefano Tiozzo.<br>
                            This master thesis produced new ascidian genomes of unprecendented quality for <i>Botryllus schlosseri</i> and <i>Polyandrocarpa zorritensis</i></p>

                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Baptiste Hilaire</h3>
                            <h4>Sequence reduction to increase the speed of bioinformatic tools</h4>
                            <p>Bachelor internship of 6 weeks (2023)<br>
                            This internship explored the properties of <a href="https://www.cell.com/iscience/pdf/S2589-0042(22)01577-2.pdf"> Mapping-friendly Sequence Reductions (MSRs) </a>, with the final objective to increase the speed and/or decrease the memory needed for some bioinformatic pipelines.

                        </div>
            </div>


        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Teaching</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Initiation to Java programming</h3>

                            <p>
                            	20hx2 of practical sessions with 1st year bachelor computer science students at the university of Rennes, Fall 2021 and Fall 2022
                            </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Python object-oriented programming</h3>
                            <h4></h4>

                            <p>
                            	34hx2 of practical sessions with 1st year master bioinformatics students at the university of Rennes, Winter 2022 and Winter 2023
                            </p>

                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Bioinformatics crash course</h3>
                            <h4></h4>

                            <p>
                            A full day crash-course (6h) of bioinformatics to high-school students selected to represent Belgium at the International Biology Olympiad, Brussels, May 2022 and 2023.
                            </p>

                        </div>
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<!-- .section-education -->

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Educational background</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        
                        <!-- .experience-item -->
                        <div class="content-item">
                            <small>2020 - 2021</small>
                            <h3>Master "Bioinformatics & modelisation"</h3>
                            <h4>Sorbonne universités</h4>

                            <p>Paris, France</p>
                        </div>
                        
                        <!-- .experience-item -->
                        <div class="content-item">
                            <small>2017-2020</small>
                            <h3>Engineering school</h3>
                            <h4>École polytechnique</h4>

                            <p>Saclay, France</p>
                        </div>
                        
                        <div class="content-item">
                            <small>2015 - 2017</small>
                            <h3>Math & Physics preparatory school</h3>
                            <h4>Lycée Aux Lazaristes</h4>

                            <p>Lyon, France</p>
                        </div>
                        <!-- .experience-item -->
                    </div>
                </div>
                <!--.row-->
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>
<!-- .section-education -->



<section class="section-contact section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title">
                    <h2>Contact</h2>
                </div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-3">
                        <address>
                            <strong>Email</strong><br>
                            <a href="mailto:#">roland.faure@irisa.fr</a>
                        </address>
                    </div>
                    <div class="col-md-3">
                        <address>
                            <strong>GitHub</strong><br>
                            RolandFaure
                        </address>
                    </div>

                </div>
                <!--.row-->


            </div>
        </div>
        <!--.row-->

    </div>
    <!--.container-fluid-->
</section>
<!--.section-contact-->

<footer class="footer">
    <div class="copyright-section">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="copytext">&copy; MyProfile, All rights reserved | Design By: <a
                            href="https://themehippo.com">themehippo</a></div>
                </div>
            </div>
            <!--.row-->
        </div>
        <!-- .container-fluid -->
    </div>
    <!-- .copyright-section -->
</footer>
<!-- .footer -->

</div>
<!-- #main-wrapper -->

<!-- jquery -->
<script src="js/jquery-2.1.4.min.js"></script>
<!-- Bootstrap -->
<script src="js/bootstrap.min.js"></script>
<!--owl carousal-->
<script src="js/owl.carousel.min.js"></script>
<!--theme script-->
<script src="js/scripts.js"></script>
</body>
</html>
