<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>About me - Roland Faure</title>


    <!-- web-fonts -->
    <link href="https://fonts.googleapis.com/css?family=Hind:300,400,500,600,700" rel="stylesheet">

    <!-- font-awesome -->
    <link href="css/font-awesome.min.css" rel="stylesheet">

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- owl carousal -->
    <link href="css/owl.carousel.css" rel="stylesheet">

    <!-- Style CSS -->
    <link href="css/style.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<body>
<div id="main-wrapper">
<!-- Page Preloader -->
<div id="preloader">
    <div id="status">
        <div class="status-mes"></div>
    </div>
</div>

<header class="header">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="profile-img">
                    <img src="img/img-profile.png" class="img-responsive" alt=""/>
                </div>
                <!-- Profile Image -->
            </div>
            <div class="col-md-9">
                <div class="name-wrapper">
                    <h1 class="name">Roland Faure</h1>
                </div>
                <p>
                   I am a postdoctoral researcher specializing in sequence bioinformatics, with a passion for designing and implementing algorithmic solutions to biological challenges. During my Ph.D., my work centered on genome assembly, particularly the complexities of haplotyping and metagenome assembly. This led to the development of two notable software tools: <a target="_blank" href="https://github.com/rolandfaure/alice-asm">Alice</a> and <a target="_blank" href="https://github.com/rolandfaure/alice-asm">HairSplitter</a>. In my current postdoctoral research, I am exploring indexation problems.
                </p>


      
            </div>
        </div>
    </div>
</header>
<!-- .header-->

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Career</h2></div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Postdoc</h3>
                            <small>2025-?</small>
                            <h4>Aligning on the SRA</h4>
                            
			    <p><i>In <a target="_blank" href="http://rayan.chikhi.name/">Rayan Chikhi</a>'s lab at the institut Pasteur, France. The goal is to search efficiently through the Logan database, in order to provide a "super-BLAST" that would work on the full SRA. </i></p>
			

                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Ph.D.</h3>
                            <small>2021-2024</small>
                            <h4>Haplotype assembly from long reads</h4>
                            
                            <p>The Ph.D. manuscript is available <a target="_blank" href="docs/these_compressed.pdf">here</a>.
                            
			    <p><i>Under the joint supervision of <a target="_blank" href="http://lavenier.net/homepage/"> Dominique Lavenier </a> in the Genscale team in Rennes, France and <a target="_blank" href="https://ebe.ulb.be/ebe/Flot.html"> Jean-François Flot</a> in the EBE team at the Université libre de Bruxelles, Belgium. The main work of my PhD was to develop methods to obtain haplotype-separated assemblies. In the realm of noisy long reads, this led to HairSplitter (see below). In the realm of highly accurate long reads, this led to the Alice assembler (see below). </i></p>
			

                        </div>
            </div>
	</div>
	</div>
	</div>
</div>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Software, publications & talks</h2></div>
            </div>
          
          <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                        
                            <h3><a target="_blank" href="https://github.com/rolandfaure/alice-asm">Alice</a></h3>
                            <h4>Extremely fast and haplotype-aware assembly of high-fidelity long reads.</h4>
			    
                            <p>
                            	Existing fast high-fidelity read assemblers (mDBG, metaMDBG) are based on k-min-mer sketches of the reads. This sketching method makes it impossible to assemble close strains. The idea of Alice is to introduce a new sketching method, MSR sketching, which does not have the same shortcoming. Alice is extremely fast while being haplotype-aware.<br>
                            	Presented as a talk at <a target="_blank" href="https://seqbim.cnrs.fr/seqbim-2023/"> SeqBIM 2023</a> and <a target="_blank" href="https://coursesandconferences.wellcomeconnectingscience.org/event/genome-informatics-20241113/"> Genome Informatics 2024 </a>, where I was granted a bursary to come and present it.
                            It was presented and discussed MSR sketching at DSB 2025 in Pisa. You can find the slides <a target="_blank" href="docs/DSB_2025.pdf">here</a>.
                            </p>

			    <p>The closest thing to a paper there exist yet is Chapter 3 of <a target="_blank" href="docs/these_compressed.pdf#page=86"> my Ph.D. thesis </a></p>


                        </div>
            </div>
            
          
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                        
                            <h3><a target="_blank" href="https://github.com/rolandfaure/hairsplitter">HairSplitter</a></h3>
                            <h4>Separating noisy long reads into an unknown number of haplotypes</h4>

			    <p> <a target="_blank" href="https://peercommunityjournal.org/item/10_24072_pcjournal_481/"> Link to the Peer-Community Journal paper </a></p>
			    
                            <p>
                            	The goal of Hairsplitter is to produce contiguous and uncollapsed (i.e. with all haplotypes still present) (meta-)genome assembly. It takes an assembly and sequencing reads as input, detects if contigs have been collapsed and output the corresponding uncollapsed assembly. Written in C++. <a target="_blank" href="https://github.com/RolandFaure/Hairsplitter" > Available on GitHub.</a> The most complete paper yet <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2024.02.13.580067v2">can be found on biorxiv</a>.<br>
                            	Presented as a poster at <a target="_blank" href="https://coursesandconferences.wellcomeconnectingscience.org/event/genome-informatics-20220921/"> Genome informatics 2022</a> and <a target="_blank" href=http://iscb.org/ismbeccb2023> ISMB/ECCB 2023 </a>. <a target="_blank" href=https://RolandFaure.github.io/docs/Faure-Roland-Genscale-ISMBECCB2023.pdf> See the poster</a>. Presented as a talk at <a target="_blank" href="https://seqbim.cnrs.fr/"> SeqBIM 2022</a>, <a target="_blank" href=https://jobim2023.sciencesconf.org> JOBIM 2023 </a> and <a target="_blank" href="http://iscb.org/ismbeccb2023"> ISMB/ECCB 2023 </a>.
                            </p>

			    <p> <a target="_blank" href="https://www.scitepress.org/Papers/2024/123207/123207.pdf"> Link to the ILP improvement paper (presented at BIOSTEC 2024) </a></p>
			    <p>
			    	An Integer Linear Programming version of HairSplitter was developed with Tam Truong and Rumen Andonov and presented at <a target="_blank" href="https://bioinformatics.scitevents.org/"> 17th International Joint Conference on Biomedical Engineering Systems and Technologies </a>, where it received the best student paper award. It is <a target="_blank" href="https://github.com/rolandfaure/strainminer"> available on github </a></p>

                        </div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3><a target="_blank" href="https://github.com/nadegeguiglielmoni/graphunzip">GraphUnzip</a></h3>
                            <h4>Untangling assembly graphs to finish assemblies</h4>
                            <p><i>Developped under the supervision of Nadège Guiglielmoni and Jean-François Flot </i></p>

                            
			    <p> <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2021.01.29.428779v1.full">Link to the bioRxiv paper </a></p>
			    <p>
                            	GraphUnzip untangles assembly graphs with the help of long reads and/or Hi-C. It is designed to distinguish the different haplotypes present. Written in Python. <a target="_blank" href="https://github.com/nadegeguiglielmoni/GraphUnzip" > Available on GitHub.</a><br>
                            	Presented the key idea for the first time at <a target="_blank" href="https://jobim2021.sciencesconf.org/"> JOBIM 2021 </a> <br>
                            	The long read algorithm was presented for the first time at <a target="_blank" href="https://seqbim.cnrs.fr/"> SeqBIM </a> 2021 thanks to a grant from the SFBI.
                            </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3><a target="_blank" href="https://github.com/rolandfaure/alice-asm">QuickDeconvolution</a></h3>
                            <h4>Rapidly separating barcoded reads into groups</h4>
                            
			    <p><i>Developped under the supervision of Dominique Lavenier </i></p>	

			    <p> <a target="_blank" href="https://academic.oup.com/bioinformaticsadvances/advance-article-abstract/doi/10.1093/bioadv/vbac068/6717790">Link to the published paper (Bioinformatics advances) </a></p>
                            <p>
                            	QuickDeconvolution aims at answering the barcode deconvolution problem when dealing with barcoded reads. It strives at being scalable and parallelisable. Written in C++. <a target="_blank" href="https://github.com/RolandFaure/QuickDeconvolution" > Available on GitHub.</a><br>
                            </p>

                        </div>
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Reviews</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                    	<div class="content-item">
				<h3>Recomb-SEQ 2024</h3>

                            <p>
                            	Reviewed a paper with the help of Claire Lemaitre, February 2024
                            </p>
                          
                        </div>
                        <div class="content-item">
				<h3>ISMB/ECCB 2024</h3>

                            <p>
                            	Reviewed a paper with the help of Dominique Lavenier, January 2024
                            </p>
                          
                        </div>
                        <div class="content-item">
                            <h3>NAR Genomics and Bioinformatics</h3>

                            <p>
                            	Co-reviewed a paper with Karel Brinda, October 2022
                            </p>
                          
                        </div>
            </div>

            </div>
            
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Students</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Tam Minh Khac Truong </h3>
				<h4>StrainMiner: Data mining and discrete optimizations for strains separation in metagenomes using long reads</h4>
				<p>Two masters internships of 4 months (2022) and 6 months (2023). Co-supervised with Rumen Andonov <br>
                            	Worked on a new approach to separate error-prone sequencing reads from several strains of the same species in metagenome. The new approach is based on integer linear programming and showed good performance. This work has been integrated into HairSplitter and presented at the<a target="_blank" href="https://bioinformatics.scitevents.org/"> 17th International Joint Conference on Biomedical Engineering Systems and Technologies </a>. It is <a target="_blank" href="https://github.com/rolandfaure/strainminer"> available on github </a>.
   </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Olivier De Thier</h3>
                            <h4>Improving ascidian de novo genome assemblies with long reads</h4>
                            <p>Master thesis of 6 months (2023). Co-supervised with Jean-François Flot and Stefano Tiozzo.<br>
                            This master thesis produced new ascidian genomes of unprecendented quality for <i>Botryllus schlosseri</i> and <i>Polyandrocarpa zorritensis</i></p>

                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Baptiste Hilaire</h3>
                            <h4>Sequence reduction to increase the speed of bioinformatic tools</h4>
                            <p>Bachelor internship of 6 weeks (2023)<br>
                            This internship explored the properties of <a target="_blank" href="https://www.cell.com/iscience/pdf/S2589-0042(22)01577-2.pdf"> Mapping-friendly Sequence Reductions (MSRs) </a>, with the final objective to increase the speed and/or decrease the memory needed for some bioinformatic pipelines.

                        </div>
            </div>


        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Teaching</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Initiation to Java programming</h3>

                            <p>
                            	20hx2 of practical sessions with 1st year bachelor computer science students at the university of Rennes, Fall 2021 and Fall 2022
                            </p>
                          
                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Python object-oriented programming</h3>
                            <h4></h4>

                            <p>
                            	34hx2 of practical sessions with 1st year master bioinformatics students at the university of Rennes, Winter 2022 and Winter 2023
                            </p>

                        </div>
            </div>
            
            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        <div class="content-item">
                            <h3>Bioinformatics crash course</h3>
                            <h4></h4>

                            <p>
                            A full day crash-course (6h) of bioinformatics to high-school students selected to represent Belgium at the International Biology Olympiad, Brussels, May 2022, 2023 and 2024.
                            </p>

                        </div>
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>

<!-- .section-education -->

<section class="section-education section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title"><h2>Educational background</h2></div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-12">
                        
                        <!-- .experience-item -->
                        <div class="content-item">
                            <small>2020 - 2021</small>
                            <h3>Master "Bioinformatics & modelisation"</h3>
                            <h4>Sorbonne universités</h4>

                            <p>Paris, France</p>
                        </div>
                        
                        <!-- .experience-item -->
                        <div class="content-item">
                            <small>2017-2020</small>
                            <h3>Engineering school</h3>
                            <h4>École polytechnique</h4>

                            <p>Saclay, France</p>
                        </div>
                        
                        <div class="content-item">
                            <small>2015 - 2017</small>
                            <h3>Math & Physics preparatory school</h3>
                            <h4>Lycée Aux Lazaristes</h4>

                            <p>Lyon, France</p>
                        </div>
                        <!-- .experience-item -->
                    </div>
                </div>
                <!--.row-->
            </div>

        </div>
        <!--.row-->
    </div>
    <!-- .container -->
</section>
<!-- .section-education -->



<section class="section-contact section-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <div class="section-title">
                    <h2>Contact</h2>
                </div>
            </div>

            <div class="col-md-9">
                <div class="row">
                    <div class="col-md-3">
                        <address>
                            <strong>Email</strong><br>
                            <a href="mailto:#">roland.faure@pasteur.fr</a>
                        </address>
                    </div>
                    <div class="col-md-3">
                        <address>
                            <strong>GitHub</strong><br>
                            RolandFaure
                        </address>
                    </div>
                    <div class="col-md-3">
                        <address>
                            <strong>Bluesky</strong><br>
                            @rfaure
                        </address>
                    </div>

                </div>
                <!--.row-->


            </div>
        </div>
        <!--.row-->

    </div>
    <!--.container-fluid-->
</section>
<!--.section-contact-->

<footer class="footer">
    <div class="copyright-section">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="copytext">&copy; MyProfile, All rights reserved | Design By: <a target="_blank"
                            href="https://themehippo.com">themehippo</a></div>
                </div>
            </div>
            <!--.row-->
        </div>
        <!-- .container-fluid -->
    </div>
    <!-- .copyright-section -->
</footer>
<!-- .footer -->

</div>
<!-- #main-wrapper -->

<!-- jquery -->
<script src="js/jquery-2.1.4.min.js"></script>
<!-- Bootstrap -->
<script src="js/bootstrap.min.js"></script>
<!--owl carousal-->
<script src="js/owl.carousel.min.js"></script>
<!--theme script-->
<script src="js/scripts.js"></script>
</body>
</html>
